README.md

```diff
@@ -48,6 +48,15 @@ Occasional UI bugs may occur – in this case, please create an issue.
 - To save JWT token, [Flutter Secure Storage](https://pub.dev/packages/flutter_secure_storage) is used
 - If you find something in code that looks a bit odd, it might be some useful extension method from `lib/extensions`
 
+#### Setup instructions
+
+- Install Flutter by following instructions from [flutter.dev](https://flutter.dev).
+- Make sure you install the **Flutter** and **Dart** plugins. The installation instructions above should have instructions on this too.
+- Fork and clone the [mentorship-flutter](https://github.com/anitab-org/mentorship-flutter) repository to your local machine.
+- Open the project on your IDE. Don't be shocked if you see error warnings.
+- Open the terminal and run `flutter channel beta` to switch from stable channel to beta channel of flutter. The beta channel is the one we are using for this project.
+- Once done, run the project. By default the backend of this project is set to the mentorship-backend dev server. You are now set to go.
```

Comment:
Can you mention how to run the project here? the commands flutter run for runnning in debug mode and the flutter build command to build a release apk?

Reply:
Atm, I only know to run the project using VSCode debugger using flutter or by running flutter run on the terminal. I never really explore the flutter build app before and don't know what it is used for. can you explain it to me so I can understand the usage @Techno-Disaster ?

Reply:
also, which options would be best to build app? when I run flutter build apk, it says 
```quote
You are building a fat APK that includes binaries for android-arm,
android-arm64, android-x64.
If you are deploying the app to the Play Store, it's recommended to use
app bundles or split the APK to reduce the APK size.
    To generate an app bundle, run:
        flutter build appbundle --target-platform
        android-arm,android-arm64,android-x64
        Learn more on: https://developer.android.com/guide/app-bundle
    To split the APKs per ABI, run:
        flutter build apk --target-platform
        android-arm,android-arm64,android-x64 --split-per-abi
        Learn more on:
        https://developer.android.com/studio/build/configure-apk-splits#c
        onfigure-abi-split
```

Reply:
Add this:
### Android
To run the app in debug mode you can run ```flutter run``` in your terminal To build a release build you can run the ```flutter build apk``` command. To build the apk specific to your device arch you can run ```flutter build apk --split-per-abi ```. You can read more about this [here](https://flutter.dev/docs/deployment/android)

### IOS
add similar commands from ios from the link above here

Reply:
so, do we need to add' --target-platform android-arm,android-arm64,android-x64' on the command?

Reply:
if you run split per avi command you wll get all 3 arch files under your release folder. if you specfy the arch in yout command you will get only that file under release folder

Reply:
I see... ok, I'll make the changes now. 😉

Reply:
@Techno-Disaster, the only thing I can see on the link, to build iOS app, we use `flutter build iOS' command. But the process itself is more complicated than Android coz it has to do with App Store account and Xcode project settings. Should I just add Android for now, coz If I don't think just adding the command `flutter build IOS` on it's own is enough to explain how to build APK release in IOS.

Reply:
we might get some good IOS developers too who want to contribute or new people with IOS. @isabelcosta what do you think should we skip this iOS part?

Reply:
I guess I can add just the build command and also give the link that says to know more on create build archive on Xcode project, release on TewstFlight or to App Store, click the link provided. 😉

Reply:
also, can I ask what is the command for running the IOS app on XCode, @Techno-Disaster? Just so we have the same instructions as per Android (run and build app commands)

Reply:
the run command is flutter run (same as andorid)
for building you can link this https://flutter.dev/docs/deployment/ios